docs/modules/converter/pages/syntax/syntax-highlighting.adoc

```diff
@@ -67,3 +67,87 @@ Alternatively, you can use http://rouge.jneen.net/[Rouge], http://coderay.rubych
 if you are using the Asciidoctor/Ruby/Bundler toolchain (not Asciidoctor.js/JavaScript/npm).
 Check the `examples/` directory for examples and notes about what needs to be done for them to work.
 They are considered unsupported by the asciidoctor-reveal.js project.
+
+== Line number highlights
+
+reveal.js can add https://revealjs.com/code/#line-numbers-%26-highlights[Line Numbers & Highlights] to source code blocks.
+
+This example `source` block renders with no line numbers by default:
+
+[source, asciidoc]
+-----
+[source,java]
+----
+public class TestClass {
+    public TestClass() {
+
+    }
+
+    public void testMethod() {
+
+    }
+}
+----
+-----
+
+By adding `linenums`, line numbers will be added:
+
+[source, asciidoc]
+-----
+[source,java,linenums]
+----
+public class TestClass {
+    public TestClass() {
+
+    }
+
+    public void testMethod() {
+
+    }
+}
+----
+-----
+
+It is also possible to highlight only part of the source by using `highlight`. This will highlight the `testMethod()` code:
```

Comment:
(suggestion)

```suggestion
It is also possible to highlight only part of the source by using `highlight`. This will highlight the whole `testMethod()` method:
```